optimizer/plan/new_plans.go

```diff
@@ -43,3 +44,46 @@ type Join struct {
 	RightConditions []ast.ExprNode
 	OtherConditions []ast.ExprNode
 }
+
+// AddChild for parent
+func AddChild(parent Plan, child Plan) {
+	if child == nil || parent == nil {
+		return
+	}
+	child.AddParent(parent)
+	parent.AddChild(child)
+}
+
+// InsertPlan means inserting plan between two plans
+func InsertPlan(child Plan, parent Plan, insert Plan) error {
```

Comment:
Change the parameter order to `parent, child, insert` would be better, which is consistent with the `AddChild` method above.